src/components/catalog/catalog-tree.vue

```diff
@@ -43,13 +47,41 @@ function toggleParent(node: LayerTreeNodeModel) {
 function toggleLayer(node: LayerTreeNodeModel) {
   layers.toggleLayer(parseInt(node.id, 10), !node.checked)
 }
+
+const { i18next } = useTranslation()
```

Comment:
Typically, all this section is a composable.
Shared in the tree and the layer manager